src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -782,15 +752,14 @@ private function getAddonsConnectToolbar()
                 'href' => '#',
                 'desc' => $addonsEmail['username_addons'],
                 'icon' => 'exit_to_app',
-                'help' => $translator->trans('Synchronized with Addons marketplace!', array(), 'Admin.Modules.Notification'),
-                'data-trololo' => 'lol',
```

Comment:
@Quetzacoalt91 I've removed this entry, I hope this is not really used :trollface:

Reply:
![NOOOOO](https://i.imgur.com/8KFAJSZ.gif)